### ISSUE TITLE
Import into Data Library from path is broken

### ISSUE THREAD

**Issue by bgruening** [galaxyproject/galaxy#15762]
The following happens if I try to import files into a DL using the "from path" function.

![grafik](https://user-images.githubusercontent.com/469983/224349065-c99e7dcf-8fa6-4372-a984-e29b9d9180eb.png)

```
Mar 10 15:50:51 sn06.galaxyproject.eu gunicorn[1808474]: galaxy.web.framework.decorators ERROR 2023-03-10 15:50:51,229 [pN:main.4,p:1968004,tN:WSGI_0] Uncaught exception in exposed API method:
Mar 10 15:50:51 sn06.galaxyproject.eu gunicorn[1808474]: Traceback (most recent call last):
Mar 10 15:50:51 sn06.galaxyproject.eu gunicorn[1808474]:   File "/opt/galaxy/server/lib/galaxy/web/framework/decorators.py", line 337, in decorator
Mar 10 15:50:51 sn06.galaxyproject.eu gunicorn[1808474]:     rval = func(self, trans, *args, **kwargs)
Mar 10 15:50:51 sn06.galaxyproject.eu gunicorn[1808474]:   File "/opt/galaxy/server/lib/galaxy/webapps/galaxy/api/library_datasets.py", line 557, in load
Mar 10 15:50:51 sn06.galaxyproject.eu gunicorn[1808474]:     uploaded_datasets_bunch = self._get_path_paste_uploaded_datasets(trans, kwd, library_bunch, 200, "")
Mar 10 15:50:51 sn06.galaxyproject.eu gunicorn[1808474]:   File "/opt/galaxy/server/lib/galaxy/actions/library.py", line 218, in _get_path_paste_uploaded_datasets
Mar 10 15:50:51 sn06.galaxyproject.eu gunicorn[1808474]:     self._make_library_uploaded_dataset(trans, params, name, path, "path_paste", library_bunch, folder)
Mar 10 15:50:51 sn06.galaxyproject.eu gunicorn[1808474]:   File "/opt/galaxy/server/lib/galaxy/actions/library.py", line 296, in _make_library_uploaded_dataset
Mar 10 15:50:51 sn06.galaxyproject.eu gunicorn[1808474]:     uploaded_dataset.data = upload_common.new_upload(trans, "api", uploaded_dataset, library_bunch)
Mar 10 15:50:51 sn06.galaxyproject.eu gunicorn[1808474]:   File "/opt/galaxy/server/lib/galaxy/tools/actions/upload_common.py", line 256, in new_upload
Mar 10 15:50:51 sn06.galaxyproject.eu gunicorn[1808474]:     upload_target_dataset_instance = __new_library_upload(
Mar 10 15:50:51 sn06.galaxyproject.eu gunicorn[1808474]:   File "/opt/galaxy/server/lib/galaxy/tools/actions/upload_common.py", line 161, in __new_library_upload
Mar 10 15:50:51 sn06.galaxyproject.eu gunicorn[1808474]:     matches = [x for x in active_folders(trans, folder) if x.name == name]
Mar 10 15:50:51 sn06.galaxyproject.eu gunicorn[1808474]:   File "/opt/galaxy/server/lib/galaxy/tools/actions/upload_common.py", line 425, in active_folders
Mar 10 15:50:51 sn06.galaxyproject.eu gunicorn[1808474]:     raise
Mar 10 15:50:51 sn06.galaxyproject.eu gunicorn[1808474]: RuntimeError: No active exception to reraise
```

**Comment by bgruening** [comment]
https://github.com/galaxyproject/galaxy/blame/a2d5f4a432b11831f9bff536ca9f1efb7627cbdb/lib/galaxy/tools/actions/upload_common.py#L425

@jdavcs any idea?

**Comment by jdavcs** [comment]
@bgruening I am sorry - this is debugging code which shouldn't be there!!

**Comment by jdavcs** [comment]
will be fixed in a min.

**Comment by jdavcs** [comment]
Fixed in #15764.
